ci-operator/config/openshift/assisted-service/openshift-assisted-service-cloud_hotfix_releases.yaml

```diff
@@ -222,8 +222,6 @@ tests:
       ASSISTED_CONFIG: |
         SERVICE_BASE_REF=cloud_hotfix_releases
         STATIC_IPS=true
-        TEST_FUNC=test_networking
-      PACKET_PLAN: m3.large.x86
```

Comment:
what's this change do?

Reply:
ohhh I haven't even realized :facepalm: 
Leftovers... sorry